packages/i18n/fr_FR.js

```diff
@@ -47,4 +53,16 @@ export const frFR = {
   'rewards.cleaningPoints': 'Points de Nettoyage',
   'rewards.creativePoints': 'Points de Créativité',
   'rewards.health': 'Vie',
+  'realm.duty': 'Royaume du Devoir',
+  'realm.fullfilment': "Royaume de l'Accomplissement",
+  'realm.artist': "Royaume de l'Artiste",
+  'realm.musician': 'Royaume de la Musique',
+  'realm.writer': "Royaume de l'Ecriture",
+  'realm.engineer': "Royaume de l'Ingénieurie",
+  'realm.cook': 'Royaume de la Cuisine',
+  'realm.gardener': 'Royaume du Jardinage',
+  'realm.healing': "Royaume de l'Apaisement",
+  'realm.physical': 'Royaume de la Physicalité',
```

Comment:
Now that I'm looking back on this, I'm kind of bothered by this translation, it kind of feels wrong.

I'm guessing it's about physical activities? like sport? If so I'd change this to `Royaume du Physique`.

Reply:
Yup, it's more for physical activities like sports, yard work, gym, etc!

Reply:
i changed it!!